client/components/drop-zone/index.jsx

```diff
@@ -251,7 +253,8 @@ export const DropZone = React.createClass( {
 			'is-active': this.state.isDraggingOverDocument || this.state.isDraggingOverElement,
 			'is-dragging-over-document': this.state.isDraggingOverDocument,
 			'is-dragging-over-element': this.state.isDraggingOverElement,
-			'is-full-screen': this.props.fullScreen
+			'is-full-screen': this.props.fullScreen,
+			[ this.props.customClass ]: true,
```

Comment:
I was trying to figure that out, but I couldn't quite figure out the `classNames` documentation on that :) Wanted to make sure it works. Thanks for pointing me to the right way!